src/views/FileEditView.vue

```diff
@@ -1,5 +1,212 @@
-<script setup></script>
+<script setup>
+import { inject, onMounted, ref } from "vue";
+import {
+  createAndUpdateFile,
+  getFileDetails,
+  getSessionDetails,
+} from "@/api/index.js";
+import { useRoute, useRouter } from "vue-router";
+import { querySessionDetailsMethod } from "@/methods/session-view/index.js";
+import { decodeString, useLoader } from "@/helpers/index.js";
+import {
+  queryFileDetailsMethod,
+  initEOXJSONFormMethod,
+  hideHiddenFieldsMethod,
+} from "../methods/file-edit-view";
+import { ActionTabFileEditor } from "@/components/file/index.js";
+import isEqual from "lodash.isequal";
+import "@eox/jsonform/dist/eox-jsonform.js";
```

Comment:
This was an issue on the eox-jsonform side which I just fixed -  if you upgrade to `v0.9.1`, you can directly use `import "@eox/jsonform` and have better debugging and type support.